megamek/src/megamek/client/bot/princess/Princess.java

```diff
@@ -1239,8 +1239,8 @@ protected MovePath continueMovementFor(final Entity entity) {
 
             final long startTime = System.currentTimeMillis();
             getPathRanker(entity).initUnitTurn(entity, getGame());
-            final double fallTolerance =
-                    getBehaviorSettings().getFallShameIndex() / 10d;
+            // fall tolerance range between 0.50 and 1.0
```

Comment:
One of the problems with the path ranking algorithm is that it doesn't normalize the "how much damage can I possibly take from enemy fire" evaluation and the "how much damage can I take from falling" evaluation. The latter is always going to be considerably lower, because you only fall over once during a move, but get shot at by many things. This is an ok stopgap for smaller matches, but will have little effect during larger fights.